### PR TITLE
Update EteSync pricing and VPN server data

### PR DESCRIPTION
**Pull request by djoate**
## Description

Resolves: #none 

#### Check List 

Updated the figures for number of servers and countries for VPNs, and updated the price of EteSync.

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: 
https://deploy-preview-1438--privacytools-io.netlify.com/providers/vpn/
https://deploy-preview-1438--privacytools-io.netlify.com/software/calendar-contacts/

### CODE REVIEW

File: _includes/sections/calendar-contacts-sync.html [hunk 1]
```diff
@@ -26,15 +26,15 @@ <h1 id="calendar_contacts" class="anchor"><a href="#calendar_contacts"><i class=
...
+  description='Many email providers also offer calendar and or contacts sync services. Refer to the <a href="/providers/email">Email Providers section</a> to choose an email provider and check if they also offer calendar and/or contacts sync.'
   website="/providers/email"
 %}
 
 {%
   include cardv2.html
   title="EteSync"
   image="/assets/img/provider/etesync.png"
-  description="<strong>EteSync</strong> is a secure, end-to-end encrypted and journaled personal information (e.g. contacts and calendar) cloud synchronization and backup for Android and any OS that supports CalDAV/CardDAV. It costs $14 per year to use, or you can host the server yourself for free."
+  description="<strong>EteSync</strong> is a secure, end-to-end encrypted, and journaled personal information (e.g. contacts and calendar) cloud synchronization and backup for Android and any OS that supports CalDAV/CardDAV. It costs $24 per year to use, or you can host the server yourself for free."
```
Comment (Mikaela) [inline _includes/sections/calendar-contacts-sync.html]: https://www.etesync.com/accounts/signup/#plan

File: _includes/sections/vpn.html [hunk 2]
```diff
@@ -13,8 +13,8 @@ <h1 id="vpn" class="anchor"><a href="#vpn"><i class="fas fa-link anchor-icon"></
...
     <h2>Mullvad <span class="badge badge-info">EUR €60/Year</span></h2>
     <p><strong>Mullvad</strong> is a fast and inexpensive VPN with a serious focus on transparency and security. They have been in operation since <strong>2009</strong>. It is the only VPN provider that currently meets our criteria for recommendation. Mullvad is based in <span class="flag-icon flag-icon-se"></span> Sweden and does not have a free trial. Visit <a href="https://mullvad.net/">mullvad.net</a> to create an account.</p>
-    <h5><span class="badge badge-success">409+ Servers</span></h5>
-    <p>Mullvad has 409 servers in 38 countries at the time of writing this page. Typically the more servers a provider offers, the better: With hundreds of servers in operation, you are far more likely to find a fast connection and a server geographically closest to you.</p>
+    <h5><span class="badge badge-success">406+ Servers</span></h5>
+    <p>Mullvad has 406 servers in 39 countries at the time of writing this page. Typically the more servers a provider offers, the better: With hundreds of servers in operation, you are far more likely to find a fast connection and a server geographically closest to you.</p>
```
Comment (Mikaela) [inline _includes/sections/vpn.html]: 408 currently https://mullvad.net/en/servers/

File: _includes/sections/vpn.html [hunk 3]
```diff
@@ -55,8 +55,8 @@ <h5><span class="badge badge-danger">Not Audited</span></h5>
       <footer class="blockquote-footer">Marc Loebekken, ProtonVPN AG Legal counsel</footer>
     </blockquote>
     <p>We will reevaluate this listing at the end of 2019 or when the aforementioned report has been published, whichever is sooner.
-    <h5><span class="badge badge-success">442+ Servers</span></h5>
-    <p>ProtonVPN has 442 servers in 33 countries at the time of writing this page. Typically the more servers a provider offers, the better: With hundreds of servers in operation, you are far more likely to find a fast connection and a server geographically closest to you.</p>
+    <h5><span class="badge badge-success">526+ Servers</span></h5>
+    <p>ProtonVPN has 526 servers in 42 countries at the time of writing this page. Typically the more servers a provider offers, the better: With hundreds of servers in operation, you are far more likely to find a fast connection and a server geographically closest to you.</p>
```
Comment (Mikaela) [inline _includes/sections/vpn.html]: Do they have the number somewhere or were you counting https://protonvpn.com/vpn-servers ?

File: _includes/sections/vpn.html [hunk 4]
```diff
@@ -76,8 +76,8 @@ <h2>IVPN <span class="badge badge-info">USD $100/Year</span></h2>
...
     <p>We will reevaluate this listing at the end of 2019 or when the aforementioned report has been published, whichever is sooner.
-    <h5><span class="badge badge-success">70+ Servers</span></h5>
-    <p>IVPN has 70 servers in 29 countries at the time of writing this page. Typically the more servers a provider offers, the better. IVPN has a decent (but not exceptional) server count that will most likely provide adequate coverage to most users.</p>
+    <h5><span class="badge badge-success">77+ Servers</span></h5>
+    <p>IVPN has 77 servers in 31 countries at the time of writing this page. Typically the more servers a provider offers, the better. IVPN has a decent (but not exceptional) server count that will most likely provide adequate coverage to most users.</p>
```
Comment (Mikaela) [inline _includes/sections/vpn.html]: It's not directly said in https://www.ivpn.net/server-locations, but like previously I am assuming it to be correct.

File: _includes/sections/vpn.html (hunk 3 above)
Comment (djoate) [reply _includes/sections/vpn.html]: My commit description has a list of sources used. For ProtonVPN, https://protonvpn.com/secure-vpn

File: _includes/sections/vpn.html (hunk 4 above)
Comment (djoate) [reply _includes/sections/vpn.html]: My commit description has a list of sources used. For IVPN, https://www.ivpn.net/facts.

Though, IVPN says that they have 44 "exit nodes" in 31 countries, with 63 OpenVPN and 14 WireGuard servers. I'm assuming we're just counting the number of servers (the old count seems to have done that), but shouldn't we ultimately count the number of exit nodes?

File: _includes/sections/vpn.html (hunk 2 above)
Comment (djoate) [reply _includes/sections/vpn.html]: It's 409 now. We should just leave it at 409.

**Comment by djoate** [comment]
Unless there is a substantial change in numbers, I won't be making any more changes

**Pull request merged**